### ISSUE TITLE
any sample result available? 

### ISSUE THREAD

**Issue by aasharma90** [DrSleep/refinenet-pytorch#1]
Hi @DrSleep 

Thanks for your code. I was wondering if you have tested this code on any dataset? If yes, could you please provide me a sample result for the same? I want to try running your code on CityScapes, but thought of checking with you first whether you've any reference results available? Thanks again!

**Comment by DrSleep** [comment]
Only tested on VOC. From readme: "the provided weights achieve 80.5% mean
IoU on the validation set in the single scale setting"

**Comment by aasharma90** [comment]
Thanks for the information. One last thing, I am assuming that the training scripts and other information can be picked from your other repo containing its light-weight version. Is that correct?

**Comment by DrSleep** [comment]
Yes, you can use training scripts for Light-Weight RefineNet.

**Comment by aasharma90** [comment]
Thanks for your help! :)
